pkg/graveler/committed/manager.go

```diff
@@ -61,7 +61,7 @@ func (c *committedManager) List(ctx context.Context, ns graveler.StorageNamespac
 	return NewValueIterator(it), nil
 }
 
-func (c *committedManager) WriteRange(ctx context.Context, ns graveler.StorageNamespace, it graveler.ValueIterator) (*graveler.RangeInfo, error) {
+func (c *committedManager) WriteRange(ctx context.Context, ns graveler.StorageNamespace, it graveler.ValueIterator, _ ...graveler.SetOptionsFunc) (*graveler.RangeInfo, error) {
```

Comment:
We have two Implementations for WriteRange and WriteMetRange- one for geaveler and one for the commitManager, only graveler's implementation uses the SetOptionsFunc, but I had to add it here so it will implement the CommittedManager interface